packages/llamaindex/src/cloud/LlamaCloudIndex.ts

```diff
@@ -132,18 +132,46 @@ export class LlamaCloudIndex {
     await this.waitForPipelineIngestion(verbose, raiseOnError);
   }
 
-  private async getPipelineId(
-    name: string,
-    projectName: string,
+  public async getPipelineId(
+    name?: string,
+    projectName?: string,
   ): Promise<string> {
     const pipelines = await PipelinesService.searchPipelinesApiV1PipelinesGet({
-      projectName,
-      pipelineName: name,
+      projectName: projectName ?? this.params.projectName,
```

Comment:
as the projectName is not unique, doesn't we need here org ID too? (please check python code)

Reply:
and please double check other uses of just projectName in this file